www/index.html.tmpl

```diff
@@ -70,6 +86,14 @@
                         var search = '{' + v + '}',
                             replace = values[v] || search;
 
+                        // Set the template text as not selectable and grey it
+                        // out to make it clear that not all the variables have
+                        // been filled
+                        if (!(v in values) || values[v] === "") {
+                            $('#text').bind('mousedown', preventSelection);
+                            $('#text').prop('readonly', true);
+                        }
+
                         // fuck JavaScript
                         content = content.split(search).join(replace);
```

Comment:
lol i forgot about this

Reply:
lol yeah, I forgot about this too until I started editing this file and saw it again